ACE/ace/SOCK_Dgram.cpp

```diff
@@ -729,56 +777,38 @@ ACE_SOCK_Dgram::make_multicast_ifaddr6 (ipv6_mreq *ret_mreq,
                   0,
                   sizeof (lmreq));
 
-#ifndef ACE_LACKS_IF_NAMETOINDEX
-  if (net_if != 0)
-    {
-      lmreq.ipv6mr_interface = ACE_OS::if_nametoindex (ACE_TEXT_ALWAYS_CHAR (net_if));
-    }
-#elif defined (ACE_WIN32)
+#if defined (ACE_WIN32) || !defined (ACE_LACKS_IF_NAMETOINDEX)
   if (net_if != 0)
     {
+#if defined (ACE_WIN32)
       int if_ix = 0;
-      bool num_if =
+      bool const num_if
```

Comment:
```suggestion
      bool const num_if =
```
we will change this line 2 characters at a time until it works

Reply:
I just wanted you both to get an opportunity to get your name on a commit.